fili-core/src/main/java/com/yahoo/bard/webservice/data/metric/LogicalMetricColumn.java

```diff
@@ -21,6 +21,16 @@ public LogicalMetricColumn(String name, LogicalMetric metric) {
     }
 
     /**
+     * Constructor.
+     *
+     * @param metric  The logical metric
+     */
+    public LogicalMetricColumn(LogicalMetric metric) {
```

Comment:
It seems weird that there isn't a constructor that simply extract the `name` from the metric itself, let me know if there is a reason for it being missing.